App/Scene/Profile/ChangePassword/styles.js

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+import { Colors, Fonts, Metrics } from '~/Theme/';
+
+const styles = StyleSheet.create({
+  container: {
+    // flex: 1,
+    // paddingHorizontal: Metrics.baseMargin,
+    backgroundColor: Colors.secondaryLight,
```

Comment:
Change this color to `white`, I want to make the backgroundColor to be fixed at `white`